pkg/service/spec_test.go

```diff
@@ -0,0 +1,37 @@
+package service
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSpecBuilder_Build(t *testing.T) {
+	scope := "scope"
+	img := []byte("imageData")
+	params := make(map[string]string)
```

Comment:
can be shortened into:
`params := map[string]string{"foo":"bar"}`

Would also be nice to change the other tests to initialize the map this way too since it looks neater?